tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/OrderEntity.cs

```diff
@@ -1,99 +1,130 @@
+                @OrganisationId,
+                @OrganisationName,
+                @OrganisationOdsCode,
+                @OrganisationAddressId,
+                @OrganisationBillingAddressId,
+                @OrganisationContactId,
+                @OrderStatusId,
+                @Created,
+                @LastUpdated,
+                @LastUpdatedBy,
+                @LastUpdatedByName
             );";
-
+
         public static async Task<string> FetchOrderDescriptionFromOrderId(string connectionString, string orderId)
-        {
+        {
             return (await SqlRunner.QueryFirstAsync<string>(connectionString, @"SELECT
-                         [Description]
+                         [Description]
                          FROM dbo.[Order]
                          WHERE OrderId = @orderId;", new { orderId }));
-        }
-
-        public static async Task<string> FetchLastUpdatedByNameFromOrderId(string connectionString, string orderId)
-        {
+        }
+
+        public static async Task<string> FetchLastUpdatedByNameFromOrderId(string connectionString, string orderId)
+        {
             return (await SqlRunner.QueryFirstAsync<string>(connectionString, @"SELECT
                          LastUpdatedByName
                          FROM dbo.[Order]
                          WHERE OrderId = @orderId;", new { orderId }));
-        }
-
+        }
+
         public static async Task<OrderEntity> FetchOrderByOrderId(string connectionString, string orderId)
         {
             return (await SqlRunner.QueryFirstAsync<OrderEntity>(connectionString, @"SELECT
                           OrderId,
-                          [Description],
+                          [Description],
                           OrganisationId,
+                          OrganisationName,
+                          OrganisationOdsCode,
                           OrderStatusId,
                           Created,
                           LastUpdated,
                           LastUpdatedBy,
                           LastUpdatedByName
-                         FROM dbo.[Order]
-                         WHERE OrderId = @orderId;", new { orderId }));
-        }
-    }
-}
+                         FROM [Order]
+                         WHERE [OrderId] = @orderId", new { orderId }));
+        }
+
+        public static async Task<ContactEntity> FetchPrimaryContactByOrderId(string connectionString, string orderId)
+        {
+            return (await SqlRunner.QueryFirstAsync<ContactEntity>(connectionString, $@"SELECT
+                        [ContactId],
+                        [FirstName],
+                        [LastName],
+                        [Email],
+                        [Phone] 
+                        FROM [CatalogueOrdering].[dbo].[Order] od join [CatalogueOrdering].[dbo].[Contact] ct on od.organisationContactId = ct.ContactId
+                        WHERE [OrderId] = @orderId", new { orderId }));
+        }
```

Comment:
I guess this might change following your conversation with @RuthBerry96 but I just wanted to request some SQL tweaks – I'm hoping to discuss SQL at our standards meeting this afternoon.

Omit the square brackets except where required (just `[Order]`)
Omit the database name
Use `AS` for aliases
Use the full syntax for joins, e.g. `INNER JOIN`
Include qualifiers for all columns
Add a terminating semicolon

For example:

```sql
SELECT ct.ContactId, ct.FirstName, ct.LastName, ct.Email, ct.Phone
  FROM dbo.[Order] AS od
       INNER JOIN dbo.Contact AS ct
       ON ct.ContactId = od.OrganisationContactId
 WHERE od.OrderId = @orderId;
```
I'd probably just use single letter table aliases here as it would be fairly clear that `o` corresponds to `[Order]` and `c` corresponds to `Contact`.

I'll try to cover the rationale for this stuff this afternoon.

Reply:
in line with what Ruth has done i have dropped the join altogether and now access the address and contact from the respective entities with a standard fetch method.